### PR TITLE
Add MapView, VideoView, WebView

### PR DESCRIPTION
**Pull request by antonBatrakov**

### CODE REVIEW

File: lib/chat/video_view.dart [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+class _PlayPauseOverlay extends StatefulWidget {
+  const _PlayPauseOverlay({Key key, this.controller}) : super(key: key);
+
+  final VideoPlayerController controller;
+
+  @override
+  __PlayPauseOverlayState createState() => __PlayPauseOverlayState();
+}
+
+class __PlayPauseOverlayState extends State<_PlayPauseOverlay> {
```
Comment (DenisovAV) [inline lib/chat/video_view.dart]: Why do you need two _ ?

File: lib/chat/video_view.dart [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  @override
+  void initState() {
+    super.initState();
+    widget.controller.addListener(() {
+      setState(() {});
```
Comment (DenisovAV) [inline lib/chat/video_view.dart:92]: Looks strange, what is the goal?

Comment (antonBatrakov) [reply lib/chat/video_view.dart:92]: The goal is to update play\pause overlay according to view player state.

File: lib/chat/video_view.dart (hunk 1 above)
Comment (antonBatrakov) [reply lib/chat/video_view.dart]: Just autogenerated, because widget have single underline and it appended to state. 
Removed redundant underline.

**Pull request merged**